.github/workflows/coverage.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
       - name: Generate code coverage
         run: |
-          cargo +nightly-2021-03-24 tarpaulin --verbose --all-features --workspace --exclude parallel --exclude parallel-dev --exclude parallel-runtime --exclude vanilla-runtime --exclude heiko-runtime --exclude pallet-loans-benchmarking  --exclude-files pallets/liquid-staking/src/benchmarking.rs --timeout 120 --out Xml
+          cargo +nightly-2021-03-24 tarpaulin --verbose --all-features --workspace --exclude parallel --exclude parallel-dev --exclude parallel-runtime --exclude vanilla-runtime --exclude heiko-runtime --exclude pallet-loans-benchmarking  --exclude pallet-liquid-staking --timeout 120 --out Xml
```

Comment:
why did you exclude the whole crate? we still need to count coverage of liquid staking pallet, but we just need to ignore the benchmarking part

Reply:
`--exclude-files` is not working. I just wanted to check whether this change would work or not

Reply:
It's fine to exclude this crate for now, please create a issue to track the context so that we can add it back later.